core/crypto/ed25519-batch/benches/ed25519_benchmarks.rs

```diff
@@ -26,7 +26,7 @@ mod ed25519_benches {
     fn verify(c: &mut Criterion) {
         let mut csprng: ThreadRng = thread_rng();
         let keypair: SigningKey = SigningKey::generate(&mut csprng);
-        let msg: &[u8] = b"";
+        let msg: &[u8] = b"aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
```

Comment:
note: this is the same message used below in batch verification

Reply:
Can this be a constant?

Also it's not obvious why it's better to have it be the same - could you please include the reason in the PR description.